packages/machine-extractor/example-site/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.0",
   "scripts": {
     "dev": "next",
-    "build": "yarn add xstate-parser-demo@latest && yarn && next build",
+    "build": "yarn add @xstate/machine-extractor@latest && yarn && next build",
     "serve": "next serve"
   },
   "dependencies": {
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
     "xstate": "^4.24.1",
-    "xstate-parser-demo": "^0.1.4",
+    "@xstate/machine-extractor": "^0.1.4",
```

Comment:
looks like this `package.json` might get updated in some future PR accidentally~ since those dependencies are not sorted